packages/contract/lib/contract/properties.js

```diff
@@ -269,6 +269,10 @@ module.exports = {
     get: function() {
       var code = this._json.deployedBytecode;
 
+      if (!code) {
+        return code;
```

Comment:
Small stylistic thing but would it be better to explicitly return `null` rather than whatever falsy value `code` happens to be? Or is whatever falsy value code is important?

Reply:
I don't know that the specific value is important, no.  If we do want to make it a single explicit thing I'd lean towards `undefined` over `null` but obviously that doesn't really matter that much.